htdocs/core/ajax/onlineSign.php

```diff
@@ -410,9 +410,9 @@
 								// A signature image file is 720 x 180 (ratio 1/4) but we use only the size into PDF
 								// TODO Get position of box from PDF template
 
-								$param['xforimgstart'] = 10;
-								$param['yforimgstart'] = (empty($s['h']) ? 240 : $s['h'] - 65);
-								$param['wforimg'] = $s['w'] / 2 - $param['xforimgstart'];
+								$param['xforimgstart'] = 110;
```

Comment:
The width of pdf may vary according to page format. So to have the start on right side we should use size of pdf /2 + offset.
It seems this is what old code did (good). But new one is using hard cided value 110 (wrong)

Reply:
Thanks for help